packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.controller.js

```diff
@@ -188,6 +190,34 @@ export default class HostingGeneralInformationsCtrl {
     return !this.hasSSLCertificate();
   }
 
+  canOrderCdn() {
+    const {
+      flushCdnState,
+      hosting: { hasCdn, offer },
+    } = this.$scope;
+
+    const isFlushCdnStateOk = flushCdnState === 'ok';
+    const isOfferStart10M = offer === 'START_10_M';
+
+    const canOrderCdn =
+      (isFlushCdnStateOk && !hasCdn && !isOfferStart10M) ||
+      (hasCdn && isFlushCdnStateOk) ||
```

Comment:
This condition can be removed as the following OR will return true regardless of `isFlushCdnStateOk`

Reply:
IMO we could push it a little bit further and simply write
`hasCdn || (isFlushCdnStateOk && !isOfferStart10M)`

But frankly I didn't wanted to rework anything about this business rule as it often leads to potential side effects (and because that's a first for me :stuck_out_tongue:)